optimum/intel/openvino/modeling_text2speech.py

```diff
@@ -187,10 +186,10 @@ class _OVModelForSpeechT5ForTextToSpeech(OVModelForTextToSpeechSeq2Seq):
 
     def __init__(
         self,
-        encoder_model: ov.Model,
-        decoder_model: ov.Model,
-        postnet_model: ov.Model,
-        vocoder_model: ov.Model,
+        encoder: ov.Model,
+        decoder: ov.Model,
+        postnet: ov.Model,
+        vocoder: ov.Model,
```

Comment:
to stay consistent with other seq2seq models  https://github.com/huggingface/optimum-intel/blob/bdeea95b61a80b9b9217fb730b3609657b55d208/optimum/intel/openvino/modeling_seq2seq.py#L320